### PR TITLE
SECOND

### PR DESCRIPTION
**Pull request by luc-boussant**

### CODE REVIEW

File: src/main/java/com/usharesoft/basicmaven/Calculator.java [hunk 1]
```diff
@@ -5,20 +5,20 @@
...
 
-    public int subtract(int numberOne, int numberTwo) {
+    public float subtract(int numberOne, int numberTwo) {
         return numberOne - numberTwo;
     }
 
-    public int divide(int numberOne, int numberTwo) {
-        return numberOne / numberTwo;
+    public float divide(int numberOne, int numberTwo) {
+        return (float)numberOne / (float)numberTwo;
```
Comment (yanngit) [inline src/main/java/com/usharesoft/basicmaven/Calculator.java]: Not formatted

**Review by yanngit — CHANGES_REQUESTED** [review]
Please format your code.
Could you also accept float as function arguments ?

**Review by yanngit — APPROVED** [review]
Looks great !

**Comment by luc-boussant** [comment]
Thank you very much for your consideration

**Pull request merged**